crates/cadenza-syntax/src/generated/snapshots/cadenza_syntax__generated__test_data__invalid_parse__array_dedent_recovery__invalid_parse_array_dedent_recovery_ast.snap

```diff
@@ -3,5 +3,5 @@ source: crates/cadenza-syntax/src/generated/test_data.rs
 expression: "foo [\nbar\n"
 ---
 [
-    [[foo, [__list__]], bar],
+    [foo, [__list__, bar]],
```

Comment:
This shouldn't change. What this means is we broke error recovery and end up consuming the entire module in the case we can't find a closing bracket. That's really not great for incremental parsing.